src/storage.cc

```diff
@@ -415,17 +421,6 @@ void Storage::EmptyDB() {
   if (!s.ok()) {
     LOG(ERROR) << "[storage] Failed to destroy db, error: " << s.ToString();
   }
-
-  // Reopen db, it is empty if succeeded destroying db
-  auto s2 = Open();
```

Comment:
The expected behavior of the request should be failed if we didn't re-open the db. Another question is wether the DestroyDB can be reenter or not since we also need to destroy the db before restoring.